.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,17 @@ for guidance on how to set up your development environment,
 run the test suite, write new integrations, and more.
 -->
 
+**2.0 Upgrade Guide notes**
```

Comment:
Should this go towards the bottom under the heading "For Datadog Employees"?

Reply:
I think it should stay on top for visibility.
I think there's a good chance we might miss it if we move it down.